### DISCUSSION TITLE
Aspect ratio (mosaic) layout

### DISCUSSION THREAD

**Discussion by quantified1995** [deckerst/aves#58] [Ideas]
One of the best but rarely seen features of gallery apps is the ability to display photos in their original aspect ratio or what some call Mosaic layout. Some apps that employ this are Google photos, Samsung gallery, and recently the gallery x app on the Play store (no tagging or metadata system, definitely not feature rich). Would it be possible to allow photos to be shown this way? Would love to see it!

**Comment by deckerst** [comment]
I do like the look of mosaic layout, but it's tricky to implement, especially if I want to keep similar performance with the current layout.

So, even though it's something I'd like to add some day, it's not going to happen any time soon.

**Comment by Donkey-Doug** [comment]
The mosaic layout in the latest release is awesome! Thank you.

**Reply by deckerst** [reply]
Thanks! Glad you like it ^^
